mingw-w64-gn/PKGBUILD

```
@@ -0,0 +1,43 @@
+
+pkgname=gn
+pkgver=1822
+pkgrel=1
+pkgdesc="GN is a meta-build system that generates build files for Ninja."
+arch=('i686' 'x86_64')
```

```suggestion
arch=('any')
```
MinGW packages use `any` arch.